proposals/new-project-join-process.md

```diff
@@ -0,0 +1,72 @@
+# KEP-748: Project Donation to Kubeflow
+
+## Summary
+This KEP outlines how OSS projects can join the Kubeflow Ecosystem.
+
+## Motivation
+As Kubeflow has become a well established ecosystem and community, several
+projects may want to join the Kubeflow ecosystem to explicitly be a part of our 
+community. 
+
+Kubeflow's goal is to cover the entire AI/ML lifecycle and new projects can help 
+address missing stages in that lifecycle.
+
+### Goals
+This goal of this process is to give clear guidelines and set expectations 
+for community members about how to be formally included into the Kubeflow Ecosystem and 
+the application process.
+
+The decision making process will be separate from the application process and is at the 
+discretion of the Kubeflow Steering Committee. The application and data provided are 
+critical for the KSC to make an informed decision that is best for the longetivity 
+of the project and community.
+
+### Non-Goals
+- Give specific recommendations for evaluating any individual project.
+- Supporting project add-ons.
+
+## Proposal
+The process to join the Kubeflow Ecosystem is intended to be simple but thorough.
+
+Project owners or maintainers will apply to join by following a four
+step process. 
+
+The process is outlined in six steps:
+
+1. Create a GitHub Issue with a Google Document outlining your proposal (please allow for commentary), the document should have a rough outline with:
+    - Authors
+    - Motivation
+    - Benefits for Kubeflow
+    - Benefits for the Project's Community
+    - Community Metrics
+    - Contributor Metrics
+    - Maintainers
+    - Migration  Plan
+    - Other Related Projects
```

Comment:
Let's get those items aligned with Background Information: https://github.com/kubeflow/community/blob/051d1f9e5fea6b09cf4c86dbde3b5a3f2e1792c1/how-to/join_kubeflow_ecosystem.md#background-information

Reply:
I see, that makes sense.